packages/react-components/src/components/chart/converters/convertSeriesAndYAxis.ts

```diff
@@ -77,13 +114,23 @@ const convertYAxis = ({ color, yAxis }: ChartStyleSettingsOptions): YAXisCompone
 /**
  * converts series and yAxis together using the same style settings
  */
-export const convertSeriesAndYAxis = (styles: ChartStyleSettingsOptions) => (dataStream: DataStream) => {
-  const series = convertSeries(dataStream, styles);
+export const convertSeriesAndYAxis = (styles: ChartStyleSettingsWithDefaults) => (datastream: DataStream) => {
+  const series = convertSeries(datastream, styles);
   const yAxis = convertYAxis(styles);
 
+  const { color, significantDigits } = styles;
+  const { data } = datastream;
+
+  const toYAxisLegend = yAxisLegendGenerator({ datastream, color, significantDigits });
+
+  const yMax = maxBy(data, dataValue);
```

Comment:
not sure if this is the same yMax we were using for TC , we do not need yMin and yMax anymore